File: systems/fleetrack/find_show_view.py
import io, sys
sys.stdout = io.TextIOWrapper(sys.stdout.buffer, encoding='utf-8')
f = open(r'C:\Users\Administrator\omnis\systems\fleetrack\index.html', encoding='utf-8')
lines = f.readlines()
f.close()

# Find showView function and API constants
for i, line in enumerate(lines):
    l = line.lower()
    if ('showview' in l and 'function' in l) or \
       ('ft_machine_reg' in l) or \
       ('const ft_' in l and 'method' in l.lower()) or \
       ('loadmachine' in l and 'function' in l):
        print(f'{i+1}: {line.rstrip()[:130]}')

File: systems/fleetrack/find_showview.js
const fs = require('fs');
const html = fs.readFileSync(
  String.raw`C:\Users\Administrator\omnis\systems\fleetrack\index.html`, 'utf8');
const lines = html.split('\n');

// Find EXACT lines with showView function definition
lines.forEach((line, i) => {
  if (line.includes('function showView') || line.includes('function showView(')) {
    console.log(`L${i+1}: ${line.trim().substring(0,100)}`);
  }
});
